src/SecurityChecker/Command/SecurityCheckerCommand.php

```diff
@@ -1,12 +1,134 @@
 <?php
-declare(strict_types=1);
+declare(strict_types = 1);
+
 /**
  * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
  * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
  */
-namespace Brancho\SecurityChecker;
 
-class SecurityCheckerCommand
+namespace SecurityChecker\Command;
+
+use SecurityChecker\Result;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Process\Exception\ProcessFailedException;
+use Symfony\Component\Process\Process;
+
+class SecurityCheckerCommand extends Command
 {
+    protected const COMMAND_NAME = 'security:check';
+
+    protected const BINARY_CHECKER = 'https://github.com/fabpot/local-php-security-checker/releases/download/v1.0.0/local-php-security-checker_1.0.0_linux_amd64';
+    protected const FILE_NAME = '/tmp/security-checker';
+    protected const FALSE_POSITIVE_ISSUE_NUMBER = 'CVE-NONE-0001';
+
+    /**
+     * @return void
+     */
+    protected function configure(): void
+    {
+        $this
+            ->setName(static::COMMAND_NAME)
+            ->setDescription('Checks security issues in your project dependencies');
+    }
+
+    /**
+     * @param \Symfony\Component\Console\Input\InputInterface $input
+     * @param \Symfony\Component\Console\Output\OutputInterface $output
+     *
+     * @return int
+     */
+    public function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $this->loadFile();
+        $this->changeFileMode();
+        $commandOutput = $this->runCommand();
+        $commandOutput = $this->markFalsePositiveResults($commandOutput);
+
+        $result = $this->createResultFromCommandOutput($commandOutput);
+
+        $output->writeln($result->getVulnerabilities());
+
+        if ($result->count() === 0) {
+            return 0;
+        }
+
+        if ($result->count() === 1 && strpos($result->getVulnerabilities(), static::FALSE_POSITIVE_ISSUE_NUMBER) !== false) {
+            $output->writeln(sprintf('<info>The issue about %s is a false positive result</info>', static::FALSE_POSITIVE_ISSUE_NUMBER));
+            $output->writeln('<info>Check https://github.com/FriendsOfPHP/security-advisories/issues/511 for details</info>');
+
+            return 0;
+        }
+
+        return 1;
```

Comment:
Please consider moving this code into individual (result processor) method dedicated to return value calculation and some output behaviour. This would clean up SecurityCheckerCommand :: execute method and significantly improve its readability.